### ISSUE TITLE
Elastic Support Diagnostics blocked by Cisco Endpoint protection

### ISSUE THREAD

**Issue by Erni** [elastic/support-diagnostics#687]
Elastic Support Diagnostics Utility is being blocked by Cisco Endpoint protection, with these results:
![cisco](https://github.com/elastic/support-diagnostics/assets/725315/c21627a6-cfce-4458-baa3-de477706fc47)

![cisco2](https://github.com/elastic/support-diagnostics/assets/725315/5eb76ca8-bb9e-46a6-b545-693498b7f2bc)

The reason it gives is the following:
```
A JAR containing a Windows PE file was found. A Java Archive (JAR) is a collection of files needed to run a Java program using the Java Runtime Environment (JRE). JARs are often intended to work across operating systems, so it is unusual to see compiled material within them. Malware may embed an executable file in a JAR to have it act as a second-stage attack, where the Java code will extract and then run the executable without the user's knowledge.
```
Why is this? Is this a real security risk?

**Comment by pickypg** [comment]
@Erni I know that this is a late response, but I suspect that this was related to the fact that we were accidentally shipping our test dependencies (and their transitive dependencies) with the distribution. Starting with `v9.1.0`, we no longer ship test dependencies.

Also, it's unclear which jar it's complaining about, so if this still triggers after that release it would be good to know. Please feel free to reopen, or create a new issue, if that pops back up.

**Comment by Erni** [comment]
Thanks a lot for the response Chris. I will try it again and let you know. Hope everything is going great!
